base/compiler/typeinfer.jl

```diff
@@ -140,6 +122,47 @@ already_inferred_quick_test(interp::AbstractInterpreter, mi::MethodInstance) =
 
 # inference completed on `me`
```

Comment:
```suggestion
# inference completed on `interp`
```
?

Reply:
This entire comment is now out of place. It should go below to `finish`.